dask/blockwise.py

```diff
@@ -17,6 +22,20 @@
 )
 
 
+def blockwise_generate_chunk(idx, chunks):
+    """Generate a local dask.array chunk"""
+    ndim = len(chunks)
+    return tuple(chunks[i][idx[i]] for i in range(ndim))
+
+
+# Dictionary of registered functions for dynamic argument
+# generation within `Blockwise`.  For now, the arguments
+# to a registered function must be msgpack serializable.
+blockwise_io_reg = {
+    "generate_array_chunk": blockwise_generate_chunk,
```

Comment:
Could you add a comment that this tag must match the same in dask/array/blockwise.py?